ska_builder.py

```diff
@@ -292,15 +294,14 @@ def main():
                 f'{version_info["release"]}{version_info["label"]}:{version_info["version"]}'
 
     if args.packages:
-        pkg_names = args.packages
+        pkg_names = sorted(args.packages)
     else:
         if args.build_list:
             with open(args.build_list) as fh:
                 pkg_names = [line.strip() for line in fh
                              if not re.match(r'\s*#', line) and line.strip()]
         else:
-            pkg_names = [str(pth.name) for pth in PKG_DEFS_PATH.glob('*') if pth.is_dir()]
-        pkg_names = sorted(pkg_names)
```

Comment:
If you want the packages sorted for most use cases that is fine, but when building from scratch from a build_list I really don't want them sorted before building (just doesn't work), so I just put the sorting in the other blocks.